testing/infra/terraform/modules/benchmark_executor/instance.tf

```diff
@@ -42,6 +52,7 @@ module "vpc" {
 resource "aws_key_pair" "worker" {
   key_name   = "${var.user_name}_worker_key"
   public_key = file(var.public_key)
+  tags       = var.tags
```

Comment:
I think all AWS resources should have the tags set in `ec2_tags`. Given that, it would be easier to define the `provider "aws"` block within this module, so all created resources have the same tags consistently. This will already happen in `testing/benchmark` since the module is consumed by that folder and we define a provider block, however, for other users, we can enforce the tag standard within the module consistently.

Reply:
I've added `ec2_tags` to all resources